package.json

```diff
@@ -7,16 +8,21 @@
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
     "big.js": "^6.2.1",
+    "gh-pages": "^4.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-icons": "^4.4.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "^5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "parcel index.html",
```

Comment:
- [ ] I'm not able to run your project locally because of a misconfiguration in the package.json. Please change back the "start" config in the "scripts". It should be: `"start": "react-scripts start"`.